torchmetrics/__init__.py

```diff
@@ -41,7 +41,7 @@
     StatScores,
 )
 from torchmetrics.collections import MetricCollection  # noqa: E402
-from torchmetrics.image import PSNR, SSIM  # noqa: E402
+from torchmetrics.image import MS_SSIM, PSNR, SSIM  # noqa: E402
```

Comment:
I wonder if we should change the name to either
`MSSSIM`
or
`MultiScaleSSIM`
We are normally trying to not use `_` in metric names.

Reply:
Oh, that's a good point. I'd personally prefer the longer version `MultiScaleSSIM`, as `MSSSIM` looks quite weird to me.